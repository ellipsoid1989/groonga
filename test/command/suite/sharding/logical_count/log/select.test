#@on-error omit
plugin_register sharding
#@on-error default

table_create Logs_20150709 TABLE_NO_KEY
column_create Logs_20150709 timestamp COLUMN_SCALAR Time

load --table Logs_20150709
[
{"timestamp": "2015-07-09 20:00:00"}
]

log_level --level debug
#@add-important-log-levels debug
#@add-ignore-log-pattern /\A\[io\]/
logical_count Logs timestamp \
  --min "2015/07/09 00:00:00" \
  --max "2015/07/09 12:00:00"
#@remove-ignore-log-pattern /\A\[io\]/
#@remove-important-log-levels debug
log_level --level notice
