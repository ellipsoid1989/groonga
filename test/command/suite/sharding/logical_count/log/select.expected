plugin_register sharding
[[0,0.0,0.0],true]
table_create Logs_20150709 TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Logs_20150709 timestamp COLUMN_SCALAR Time
[[0,0.0,0.0],true]
load --table Logs_20150709
[
{"timestamp": "2015-02-05 13:49:00"}
]
[[0,0.0,0.0],1]
log_level --level debug
[[0,0.0,0.0],true]
logical_count Logs timestamp   --min "2015/07/09 00:00:00"   --max "2015/07/09 12:00:00"
[[0,0.0,0.0],1]
#|d| [logical_count][select] <Logs_20150709>: no range index
log_level --level notice
[[0,0.0,0.0],true]
